docs/docs/aztec/smart_contracts/functions/attributes.md

```diff
@@ -178,16 +178,14 @@ This macro inserts a check at the beginning of the function to ensure that the c
 assert(context.msg_sender() == context.this_address(), "Function can only be called internally");
 ```
 
-## Custom notes #[note]
+## Implementing notes #[note]
```

Comment:
Decided against using the term custom notes in docs as now we have a `#[custom_note]` macro that means a different thing - not a note you implement yourself but a note where you implement the `NoteHashing` interface on your own.